lib/rest/publizon/local-adapter/mutator/fetcher.ts

```diff
@@ -21,7 +21,7 @@ export const fetcher = async <ResponseType>({
 
   const body = data ? JSON.stringify(data) : null
   const serviceUrl = getRestServiceUrlWithParams({
-    baseUrl: "https://pubhub-openplatform.test.dbc.dk",
+    baseUrl: `${process.env.NEXT_PUBLIC_APP_URL}/pubhub`,
```

Comment:
I think we should define the route in our route config in `lib/config/resolvers/routes.ts`